arcgis-ios-sdk-samples/Route and directions/Offline routing/README.md

```diff
@@ -32,7 +32,7 @@ Tap near a road to start adding a stop to the route, tap again to place it on th
 
 ## Offline data
 
-The data used by this sample is available on [ArcGIS Online](https://arcgisruntime.maps.arcgis.com/home/item.html?id=567e14f3420d40c5a206e5c0284cf8fc).
+The data used by this sample is available on [ArcGIS Online](https://arcgis.com/home/item.html?id=567e14f3420d40c5a206e5c0284cf8fc).
```

Comment:
It is interesting to see that both `Offline routing` and `Offline geocode` use the same `streetmap_SD` tile cache/tpkx, but their actual AGOL portal items are different. I'm a bit confused here but assume it is expected. Can you explain a bit here? Thanks.

Reply:
This portal item is a zip folder containing the `streetmap_SD.tpkx` and 2 other resources that are specific to Offline routing. The portal item in Offline geocode is only the TPKX since the sample doesn't require the other data.

Reply:
I see. I'll add some note here for the Offline routing sample

- The item https://arcgis.com/home/item.html?id=567e14f3420d40c5a206e5c0284cf8fc isn't on-demand downloaded by the app.
- `sandiego.geodatabase` and `sandiego.tn`, which by description are included in this item, are actually included in the app bundle
- Therefore, this sample depends on both the tpkx and the 2 files above. Only the tpkx is downloaded from AGOL while the other 2 files in the bundle are from Apple's server via on-demand resource tag.